.github/dependabot.yml

```diff
@@ -0,0 +1,10 @@
+version: 2
+updates:
+  - package-ecosystem: "github-actions"
+    directory: "/"
+    schedule:
+      interval: "monthly"
+    groups:
```

Comment:
```suggestion
    commit-message:
      prefix: MAINT
    groups:
```

Reply:
Goodness gracious, I overlooked the commit message section of the contributing guide and only looked at the checklist. I'll update the commit and PR. Thanks for catching this!

Reply:
I think this is addressed now. Thanks!